lib/cli.js

```diff
@@ -62,7 +63,7 @@ module.exports = function(program) {
         return returnArgs;
     }
 
-    if (!args.length && process.stdin.isTTY) {
+    if (!args.length && process.stdin.isTTY && !program.autoConfigure) {
```

Comment:
as @markelog pointed out, this seems wrong. You still need to call autoconfigure on a set of files.